observation/bf_phaseup.py

```diff
@@ -114,6 +114,8 @@ def calculate_corrections(G_gains, B_gains, delays, cal_channel_freqs,
 # Set default value for any option (both standard and experiment-specific options)
 parser.set_defaults(observer='comm_test', nd_params='off', project_id='COMMTEST',
                     description='Phase-up observation that sets F-engine gains')
+parser.add_option('--max_gap_MHz', type='float', default=64.0,
+                  help='Override max_gap_MHz (default=%default)')
```

Comment:
Something like: "The maximum gap in the bandpass gain that will still be interpolated across, in MHz (default=%default)"

Reply:
Thanks. Done.